src/domRunner.js

```diff
@@ -156,48 +156,56 @@ async function generateScreenshots(
     const staticPackage = prerender
       ? undefined
       : await createStaticPackage({
-          tmpdir,
+        tmpdir,
+        publicFolders,
+      });
+
+    const results = [];
+    for (const name of targetNames) {
```

Comment:
if you're worried about performance, i'd avoid `for..of` - the iteration protocol is slow. a regular for loop should work here?
```suggestion
    for (let i = 0; i < targetNames.length; i += 1) {
      const name = targetNames[i];
```

Reply:
We are iterating over around 1 to 6 items here, so the performance of this loop is not very important.